azure-pipelines.yml

```diff
@@ -195,13 +174,41 @@ stages:
       inputs:
         targetPath: $(build.artifactstagingdirectory)/NugetPackages
         artifactName: NugetPackages
- 
+
     - task: PublishPipelineArtifact@1
       displayName: 'Publish Artifact: drop'
       inputs:
         targetPath: $(build.artifactstagingdirectory)/publish
         artifactName: drop
 
+  - job: 'BuildDacpac'
+    pool:
+      name: 'DAS - Continuous Integration'
+      demands:
+      - Agent.OS -equals Windows_NT
```

Comment:
Is this correct? I thought the build was running on Linux build agents now

Reply:
That's just for the dacpac build part which is still built on the NT box as per the devops standard/suggestion.  This approach was provided by devops.